KinWallet/View Controllers and Co./More/HelpCenterViewController.swift

```diff
@@ -19,13 +19,37 @@ private struct MessageNames {
     static let pageLoaded = "pageLoaded"
     static let isPageHelpfulSelection = "isPageHelpfulSelection"
     static let formPageLoaded = "formPageLoaded"
+    static let feedbackSubmitted = "feedbackSubmitted"
 }
 
 final class HelpCenterViewController: WebViewController {
-    private let helpCenterURL = URL(string: "https://s3.amazonaws.com/kinapp-static/faq2/index.html")!
+    enum Page: String {
+        case faq = "https://s3.amazonaws.com/kinapp-static/faq2/index.html"
+        case feedback = "https://s3.amazonaws.com/kinapp-static/faq2/support/feedback.html"
+        case support = "https://s3.amazonaws.com/kinapp-static/faq2/support/contact-us.html"
+    }
+    enum Category: String {
+        case backup_restore = "Backup %26 Restore your Kin"
+        case other = "Other"
+    }
+
+    enum SubCategory: String {
+        case other = "Other"
+        case onboarding = "On-boarding error"
+    }
 
     private let activityIndicatorView = UIActivityIndicatorView(style: .white)
     private var errorCount = 0
+    private var pageURL = URL(string: Page.faq.rawValue)!
+
+    func setPageToLoad(page: Page) {
+        pageURL = URL(string: page.rawValue)!
+    }
+
+    func setPageToLoad(page: Page, category: Category, subCategory: SubCategory) {
```

Comment:
We could merge these two methods, by setting the second and third parameters as optional, defaulting to `nil`:

` func setPageToLoad(page: Page, faqInfo: (category: Category, subCategory: SubCategory)? = nil)`